_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,441 @@
+
+The identified web content does not have to be positioned on the same web page and doesn't need to be linked to from the tested web page.
+
+- [WCAG 2.1 - Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
+- [WCAG 2.1 - Understanding Success Criterion 1.3.3: Sensory Characteristics](https://www.w3.org/WAI/WCAG21/Understanding/sensory-characteristics.html)
+- [G96: Providing textual identification of items that otherwise rely only on sensory information to be understood](https://www.w3.org/WAI/WCAG21/Techniques/general/G96)
+- [F14: Failure of Success Criterion 1.3.3 due to identifying content only by its shape or location](https://www.w3.org/WAI/WCAG21/Techniques/failures/F14)
+- [F26: Failure of Success Criterion 1.3.3 due to using a graphical symbol alone to convey information](https://www.w3.org/WAI/WCAG21/Techniques/failures/F26)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This paragraph includes the [visual reference word][] "right". The content in the second column is identified with the word "right" but also identified by referencing the word "howdy", thus matching the (**non-visual reference**) condition.
+
+```html
+<head>
+	<title>Passed example 1 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Click "howdy" on the right, for a surprise</p>
+		</div>
+		<div class="col">
+			<button onclick="alert('Surprise!')">Howdy</button>
+		</div>
+	</div>
+</body>
+```
+
+#### Passed Example 2
+
+This paragraph includes the [visual reference word][] "box". The button is identified with the word "box" but also identified by referencing that the content can be found below this content in the DOM order. Note that "below" is also a visual reference word but in this case it can also be accurately interpreted as "further in the DOM tree order" which does not rely on visual attributes alone. Hence, the paragraph matches the (**non-visual reference**) condition.
+
+```html
+<p>Interact with the box below this paragraph, for a surprise</p>
+<button onclick="alert('Surprise!')">Howdy</button>
+```
+
+#### Passed Example 3
+
+This paragraph includes the [visual reference word][] "right". The visual reference made by the word "right" is complemented by the non-visual reference made by the word "menu" to the content identified by the "Menu" heading, thus matching the (**non-visual reference**) condition.
+
+```html
+<head>
+	<title>Passed example 3 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
```

Comment:
Actually from the screenreader perspective if we mention a more clear instruction it would be easy for them to navigate to the specific component such as headings. 
In addition, G96 [Example 2](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G96#G96-ex2) expressed an example in a similar way where "heading" is specifically used to make the instruction more clear and descriptive.

Reply:
You are correct that adding more information would make it easier to navigate. However these examples should show the "bare minimum" to pass these SC to be most effective at making sure everybody evaluates in the same manner correct? Would you agree that "menu" is enough to pass the SC because it does not, technically speaking, "rely solely on sensory characteristics"?

Reply:
As being a minimum requirement, I don't have any objection with "menu".